generate-docs/script-inputs/local-repo-snippets.yaml

```diff
@@ -11227,8 +11226,7 @@ Visio.ShapeDataItemCollection.load:
         var shapeDataItems = shape.shapeDataItems;
         shapeDataItems.load();

```

Comment:
It looks like we're only reading the label and value properties after the sync, so we should load only those, instead of having an empty load.